### PR TITLE
Feature/pam 4344 change from context to redux update

### PR DESCRIPTION
**Pull request by jjtg-nav**
Just a few changes: 
- Fixed sagas so they fetch data properly.
- Removed custom ContextProviders
- Fixed BFF distribution endpoint
- Added chevron animation on button
- Removed `express-http-proxy` in favor for the much simpler `http-proxy-middleware`

To run the application through the Express server you can run `npm run build` first and then `npm run start-express`. The application will then be available on port `9000`. 

TODO: 
- Create a script that watches changes and re-compiles react-app

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -9,12 +9,25 @@ import {CandidateSaga} from './CandidateReducer'
...
+            AdReducer,
+            CandidateReducer
+        }),
+        composeWithDevTools(applyMiddleware(saga))
+    );
+
     function* rootSaga () {
-        yield all([spawn(AdSaga,CandidateSaga)])
+        yield all([
+            spawn(AdSaga),
```
Comment (jjtg-nav) [inline src/store/index.js:24]: Each spawn function can only take one saga at a time.

File: src/App.css [hunk 2]
```diff
@@ -35,18 +35,45 @@
...
+
+
+  border-radius: 5px;
+  display: block;
+  margin: 10% auto 5%;
+  position: relative;
+}
+
+.chart-button::after {
+  content: '';
```
Comment (jjtg-nav) [inline src/App.css:57]: Just a quick little animation that gives users a hint of the button being an expandable panel.

File: server/server.js [hunk 3]
```diff
@@ -30,53 +32,25 @@ console.log('BACKEND_ENDPOINT: ' + pathProperties.BACKEND_ENDPOINT )
...
-  }),
-);
+const adProxy = createProxyMiddleware('/api/ad', {
+    target: `${pathProperties.BACKEND_URL}`,
+    pathRewrite: {'^/api/ad': '/ad'}
+});
+const candidateProxy = createProxyMiddleware('/api/candidate', {
+    target: `${pathProperties.BACKEND_URL}`,
+    pathRewrite: {'^/api/candidate': '/candidate'},
+});
```
Comment (jjtg-nav) [inline server/server.js:42]: I think this is much easier to understand that the other library.

**Review by catriont — APPROVED** [review]
Ser kjempebra ut! 🎉

**Pull request merged**